### PR TITLE
Added web-optimized SVG logo

### PR DESCRIPTION
**Pull request by lzap**
The SVG original is huge, I renamed it to `foreman_original.svgz` and created
two optimized versions for web - one is deflated, one not. I used this tool to
create these:

https://github.com/scour-project/scour

Results are quite good:
- 90K foreman_original.svg
- 87K foreman.svg (only logo without logotypes)
- 16K foreman-web.svg
-  4K foreman-web.svgz

### CODE REVIEW

File: logo/foreman.svg [hunk 1]
```diff
@@ -0,0 +1,2551 @@
...
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   inkscape:export-ydpi="90.000000"
+   inkscape:export-xdpi="90.000000"
+   inkscape:export-filename="/home/jimmac/Desktop/wi-fi.png"
+   width="400"
```
Comment (domcleal) [inline logo/foreman.svg]: The canvas size should probably be reduced to surround the logo in the same way as foreman.png, at the moment it's rectangular and contains blank space on two sides.

**Comment by domcleal** [comment]
> 16K foreman-web.svg
> 4K foreman-web.svgz

I'd suggest removing one, no need to keep the same file twice.

File: logo/foreman.svg (hunk 1 above)
Comment (lzap) [reply logo/foreman.svg]: Good observation, it saved 1K (there was a transparent object in the top I missed).

**Comment by lzap** [comment]
Pushed, removed SVGZ version, it looks like support in browsers is poor anyways (only Chrome). We will use SVG version and modify Apache configuration to deflate it.

**Comment by domcleal** [comment]
Thanks @lzap.

**Pull request merged**